src/rf/apps/workers/process.py

```diff
@@ -7,10 +7,15 @@
 import apps.core.enums as enums
 from django.conf import settings
 
+import os
 import time
+import uuid
 import json
 import boto.sqs
 from boto.sqs.message import Message
+from boto.s3.key import Key
```

Comment:
BTW, I'm working on the thumbnail generation step, and the code I've written uses boto3. I could change it to boto, but I have a small preference towards using boto3.